notify/shell/shell.go

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2022, MegaEase
+ * All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package shell
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/megaease/easeprobe/global"
+	"github.com/megaease/easeprobe/notify/base"
+	"github.com/megaease/easeprobe/report"
+
+	log "github.com/sirupsen/logrus"
+)
+
+// NotifyConfig is the config for shell notify
+type NotifyConfig struct {
+	base.DefaultNotify `yaml:",inline"`
+
+	Cmd  string   `yaml:"cmd"`
+	Args []string `yaml:"args"`
+}
+
+// Config is the config for shell probe
+func (c *NotifyConfig) Config(gConf global.NotifySettings) error {
+	c.NotifyKind = "shell"
```

Comment:
"shell" appears multiple times,suggest to to declare as a constant string

Reply:
actually, it only appears once.